includes/class-wcpdf-install.php

```diff
@@ -404,7 +404,18 @@ protected function upgrade( $installed_version ) {
 				}
 			}
 		}
-		
+
+		// 3.0.0-dev-1: remove saved option 'use_html5_parser'
+		if ( version_compare( $installed_version, '3.0.0-dev-1', '<' ) ) {
+			// removes 'HTML5 parser' setting value
+			$debug_settings = get_option( 'wpo_wcpdf_settings_debug', array() );
+			if ( ! empty( $debug_settings['use_html5_parser'] ) ) {
+				unset( $debug_settings['use_html5_parser'] );
+				update_option( 'wpo_wcpdf_settings_debug', $debug_settings );
+			}
+			// force copy fonts merge to replace the fonts cache file (Dompdf <2.0)
+			WPO_WCPDF()->main->copy_fonts( $font_path, true );
```

Comment:
as you correctly pointed out in previous comments, this is already done earlier in this method:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/de3f055345804a78a51fa043ba01333e0a00fd50/includes/class-wcpdf-install.php#L218
so we don't need to repeat it here.